src/components/atoms/authors/authors.test.tsx

```diff
@@ -31,6 +31,12 @@ describe('authors', () => {
     expect(expansionElement).not.toBeInTheDocument();
   });
 
+  it.each(authors.map((author) => createAuthorId(author)))('should contain a link with the author id', (id) => {
```

Comment:
Feels like we should be mocking the `createAuthorId` function.

Reply:
@will-byrne Your thoughts on @nlisgo comments on tests?

Reply:
it does make the tests dependant on the `createAuthorId` working as expected, as this isnt an integration test mocking it is a good idea.